example/testapp/apitest/features/support/feature_hooks.js

```diff
@@ -7,15 +7,15 @@ module.exports = function () {
   var self = this;
 
   this.registerHandler('BeforeFeatures', function (event, done) {
-    childProcess.exec("../src/serv")
+    this.serverPID = childProcess.exec("../src/serv")
```

Comment:
the `this` keyword is bound to the global object here. maybe `self.serverPID` so the process is attached to the module?